src/elements/hx-beacon/index.js

```diff
@@ -0,0 +1,53 @@
+import { HXElement } from '../../interfaces/HXElement/index.js';
+
+import shadowMarkup from './_shadow.html';
+import shadowStyles from './_shadow.scss';
+
+/**
+ * Defines behavior for the `<hx-beacon>` element.
+ *
+ * @extends HXElement
+ * @emits beacon:dismiss
+ * @hideconstructor
+ * @since 0.22.0
+ */
+export class HXBeaconElement extends HXElement {
+    static get is () {
+        return 'hx-beacon';
+    }
+
+    static get template () {
+        return `<style>${shadowStyles}</style>${shadowMarkup}`;
+    }
+
+    $onCreate () {
+        this._onDismiss = this._onDismiss.bind(this);
+    }
+
+    $onConnect () {
+        this._btnDismiss.addEventListener('click', this._onDismiss);
+    }
+
+    $onDisconnect () {
+        this._btnDismiss.removeEventListener('click', this._onDismiss);
+    }
+
+    /**
+     * Dismiss the beacon (removes element from the DOM)
+     */
+    dismiss () {
+        if (this.$emit('dismiss')) {
+            this.remove();
+        }
+    }
```

Comment:
Please see if you can add component tests to `src/elements/hx-beacon/index.spec.js` for these methods.